Pipfile

```diff
@@ -33,6 +33,7 @@ greenlet = "==0.4.16"
 typing-extensions = "*"
 types-pyyaml = "*"
 types-requests = "*"
+pygithub = "*"
```

Comment:
this dependency should be pinned. 
it is typical to not pin typings (but that is perhaps debatable too), but regular libraries should be.